### DISCUSSION TITLE
Can this pin an existing hash?

### DISCUSSION THREAD

**Discussion by AndrewKvalheim** [n0-computer/iroh#692] [Q&A] [answered]
I’m looking for a lightweight service to pin existing content that was originally hashed by go-ipfs. I see that iroh can fetch these and write them to disk. Is there a way to make it pin them as well?

**Comment by DougAnderson444** [comment, marked as answer]
I was looking for the same thing, it appears that is not available yet according to https://iroh.computer/docs/iroh-and-kubo/:

![image](https://user-images.githubusercontent.com/6699901/217838520-de80553a-67ce-4350-9909-8c06967fd99b.png)
